docs/resources/gateway.md

```diff
@@ -306,7 +306,29 @@ settings {
 ```
 
 ~> **Note on GWLB Acceptance Required**
-The GWLB Acceptance required default is set to `false` when Valtix orchestrates the GWLB. In the case where a user will configure principals and control Endpoint connection acceptance using the AWS Console or AWS Terraform Provider it is desired for the Acceptance required to be set to `true`.
+The GWLB Acceptance required default is set to `false` when Valtix orchestrates the GWLB. In the case where a user will configure Principals and/or control Endpoint connection acceptance using the AWS Console or AWS Terraform Provider it is desired for the Acceptance required to be set to `true`.
+
+### To not create an AWS GWLB Endpoint (GLWBe) into the Datapath Subnet
+```hcl
+settings {
+  name  = "controller.gateway.aws.gwlb.deploy_gwlb_endpoints"
+  value = false
+}
+```
+
+~> **Note on GWLB Endpoint (GWLBe) Creation**
+The GWLB Endpoint (GWLBe) creation default is set to `true` when Valtix orchestrates the GWLB when deploying an Egress Gateway. When not specified or explicitly set to `true`, Valtix orchestrates a GWLBe and connects it to the GWLB.  When set to `false`, Valtix will not orchestrate a GWLBe and will rely on the user to create any GWLBes using the AWS Terraform Provider or AWS Console and connect them to the GWLB.
+
+### To specify GWLB Service Principals
+```hcl
+settings {
+  name  = "controller.gateway.aws.gwlb.service_principals"
+  value = ["*"]
+}

```

Comment:
@johnvaltix , can you update the example , syntax is not correct , here is the correct example : 
```
  settings {
  name  = "controller.gateway.aws.gwlb.service_principals"
  value = <<EOT
           *
           arn:aws:iam::902505820618:root
           arn:aws:iam::059126560514:root
           EOT
}
```

Reply:
@hardikvaltix @johnvaltix can you please confirm this works. Ideally the EOT on the last line must not have any spaces (no indentation). But please check it works

Reply:
@kiranvaltix  yes its works fine , there is no space